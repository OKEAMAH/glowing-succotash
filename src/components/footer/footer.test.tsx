import { render } from '@testing-library/react'

import Footer from './footer'

test('Footer renders', () => {
  const { container } = render(<Footer />)
  const footer = container.firstElementChild!

  expect(footer).toMatchSnapshot()
})
