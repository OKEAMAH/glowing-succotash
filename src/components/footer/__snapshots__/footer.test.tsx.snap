// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders 1`] = `
<footer
  class="footer footer-center p-2 text-base-content"
>
  <div>
    <p>
      <span>
        Made with ðŸ’– by 
      </span>
      <a
        class="link link-accent no-underline"
        href="https://cryogenicplanet.tech"
        target="_blank"
      >
        CryogenicPlanet
      </a>
      Â andÂ 
      <a
        class="link link-accent no-underline"
        href="https://github.com/wei/"
        target="_blank"
      >
        Wei
      </a>
    </p>
  </div>
  <a
    class="absolute bottom-2 right-2"
    href="https://vercel.com/?utm_source=github-socialify&utm_campaign=oss"
    target="_blank"
  >
    <img
      alt="Powered by Vercel"
      height="33"
      src="https://user-images.githubusercontent.com/5880908/218415645-ac47f9ba-0d60-47eb-950c-ed10bba1e7f6.png"
      width="159"
    />
  </a>
</footer>
`;
